app/components/SelectedAlertShortcutLink.tsx

```diff
@@ -0,0 +1,52 @@
+import { Link } from '@remix-run/react'
+
+import {
+  JsonNoticeTypes,
+  NoticeTypes,
+} from './NoticeTypeCheckboxes/NoticeTypeCheckboxes'
+import { WithCredentials, useEmail } from '~/root'
+
+export default function SelectedAlertShortcutLink({
+  alertKey,
+  format,
+  otherAlerts,
+  destination,
+}: {
+  alertKey: string
+  format: 'json' | 'text'
+  destination: 'quickstart' | 'email'
+  otherAlerts?: string[] // Alerts specifically under the 'Other' tab, since they can't be generically selected as all
+}) {
+  const userCredentials = WithCredentials()
+  const userEmail = useEmail()
+
+  const selectedAlerts =
+    alertKey == 'Other'
+      ? otherAlerts?.map((alert) => `&alerts=${alert}`).join('')
+      : (format == 'text' ? NoticeTypes[alertKey] : JsonNoticeTypes[alertKey])
+          ?.map((alert) => `&alerts=${alert}`)
+          .join('')
```

Comment:
Please construct the URL search params using [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) rather than with string formatting, so that there is not question of whether we have correctly escaped everything.

Reply:
Addressed in latest commit